data/eva_model_utils.py

```diff
@@ -0,0 +1,72 @@
+import torch
```

Comment:
A general comment on this file is that I think it should be renamed to 'dep_eval_utils.py' or something like this, because it is only relevant for dep. 
Also, I think it should be outside of the 'data' dir, since it is connected to evaluation and not data preparation.